src/base/bittorrent/session.cpp

```diff
@@ -5158,7 +5166,8 @@ void Session::handleSessionStatsAlert(const lt::session_stats_alert *p)
     const auto calcRate = [interval](const qint64 previous, const qint64 current) -> qint64
     {
         Q_ASSERT(current >= previous);
-        return ((current - previous) / interval);
+        Q_ASSERT(interval >= 0);
+        return (((current - previous) * 1000000) / interval);
```

Comment:
It would also be possible to use a named constant:
```suggestion
        return (((current - previous) * lt::microseconds(1s).count()) / interval);
```